cfme/configure/settings.py

```diff
@@ -20,154 +22,189 @@ class TimeProfileAddForm(View):
     timezone = BootstrapSelect('profile_tz')
     days = BootstrapSwitch(name='all_days')
     hours = BootstrapSwitch(name='all_hours')
-    save_button = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
-    configuration = Dropdown('Configuration')
-    table = Table("//div[@id='main_div']//table")
-    save_edit_button = Button('Save')
+    add = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
     cancel_button = Button('Cancel')
     help_block = Text("//span[contains(@class, 'help-block')]")
 
 
-class TimeprofileAddEntities(View):
+class TimeProfileEntities(View):
+    table = Table("//div[@id='main_div']//table")
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
 
 
-class TimeProfileAddFormView(BaseLoggedInPage):
-    timeprofile_form = View.nested(TimeProfileAddForm)
-    entities = View.nested(TimeprofileAddEntities)
+class TimeProfileView(BaseLoggedInPage):
+    form = View.nested(TimeProfileAddForm)
+    entities = View.nested(TimeProfileEntities)
     mysetting = View.nested(MySettingsView)
+    configuration = Dropdown('Configuration')
 
+    @property
+    def is_displayed(self):
+        return self.entities.title == 'Time Profile Information'
 
-class TimeProfileAllView(MySettingsView):
-    table = Table("//div[@id='main_div']//table")
-    help_block = Text("//span[contains(@class, 'help-block')]")
-    configuration = Dropdown('Configuration')
+
+class TimeProfileAllView(TimeProfileView):
 
     @property
     def is_displayed(self):
-        return self.tabs.time_profile.is_active()
+        return self.mysetting.tabs.time_profile.is_active()
 
 
-class Timeprofile(Updateable, Navigatable):
-    def __init__(self, description=None, scope=None, days=None, hours=None, timezone=None,
-                 appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.description = description
-        self.scope = scope
-        self.days = days
-        self.hours = hours
-        self.timezone = timezone
+class TimeProfileEditView(TimeProfileView):
```

Comment:
**REQUIRED**: As we have a form for Edit, I don't agree with having the Add form under all, I know it's a little verbose, but let's have a form for Add, and Edit. I agree we don't need one for Copy, but we do need a navigation destination like you already have.